### PR TITLE
Add QLVideo

### PR DESCRIPTION
**Pull request by kdeldycke**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -115,6 +115,15 @@ Run `brew cask install suspicious-package` or [download manually](http://www.mot
 [![](screenshots/SuspiciousPackage.png)](http://www.mothersruin.com/software/SuspiciousPackage/)
 
 
+### [QLVideo](https://github.com/Marginal/QLVideo)
+
+> Preview most types of video files, as well as their thumbnails, cover art and metadata
+
+Run `brew cask install qlvideo` or [download manually](https://github.com/Marginal/QLVideo/releases/latest)
+
+[![](https://raw.githubusercontent.com/Marginal/QLVideo/master/img/preview.jpeg)](https://github.com/Marginal/QLVideo)
```
Comment (sindresorhus) [inline readme.md]: The screenshot needs to be included in this repo like the others and should be the same size as the existing ones and with shadow.

Comment (kdeldycke) [inline readme.md]: Done in 440e376c18099e67d8897d30859bfac8c5d8e650 .

**Comment by kdeldycke** [comment]
All concerns have been addressed. This PR is ready to be merged.

**Comment by kdeldycke** [comment]
Thanks @sindresorhus ! :)

**Pull request merged**